src/packets.c

```diff
@@ -790,9 +790,9 @@ TbBool process_dungeon_control_packet_dungeon_control(long plyr_idx)
                 output_message(SMsg_WorkerJobsLimit, 500, true);
               }
             } else
-            if ((player->field_455 == P454_Unkn3) && ((player->field_3 & Pf3F_Unkn01) != 0))
+            if ((player->secondary_cursor_state == CSt_PowerHand) && ((player->field_3 & Pf3F_nothing_to_touch_under_cursor) != 0))
```

Comment:
field_3 = pointing_at ?

Reply:
not sure, if you look at my guesses for Pf3F_Unkn04, Pf3F_Unkn08 and Pf3F_Unkn10, it seems this group is more of a mixed bag....

Reply:
not so sure. 
Pf3F_Unkn04 -> lightning target.
Only if you correctly target something with the cursor, will the pallet swap.
Pf3F_Unkn10 -> this is something we recently did, so could be dumb. Also you could click past a screen or something?

Reply:
`field_3` is now `additional_flags` and value names cleared up